src/main/java/io/jenkins/plugins/audit/config/AuditLogRootAction.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.audit.config;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.model.DirectoryBrowserSupport;
+import hudson.model.RootAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.*;
+
+
+@Extension
+public class AuditLogRootAction implements RootAction {
+    @Override
+    public String getIconFileName() {
+        return "notepad.png";
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Audit Logs";
+    }
+
+    @Override
+    public String getUrlName() {
+        return Jenkins.get().getRootUrl() + "auditLog" ;
+    }
+
+    public DirectoryBrowserSupport doAuditLog(StaplerRequest req, StaplerResponse rsp) {
```

Comment:
```suggestion
    public DirectoryBrowserSupport doAuditLog() {
```

These parameters shouldn't be necessary.